### ISSUE TITLE
ValueError: cannot insert subsegment-idx, already exists

### ISSUE THREAD

**Issue by petiatil** [m-bain/whisperX#176]
Edit: As this error is uncommon and occurs only when processing a certain portion of the audio, I'm open to any applicable solution that skirts the error

I'm testing various English models (e.g. "small.en", "medium.en", and the "large-v2" model) and continue to get the ValueError: "cannot insert subsegment-idx, already exists" for a particular file (a large audio file with various speakers).

The error occurs in the third line:

```
result = model.transcribe(audio=audio_path, verbose=True, language="en")
model_a, metadata = whisperx.load_align_model(language_code=result["language"], device=device)
result_aligned = whisperx.align(result["segments"], model_a, metadata, audio_path, device)
```

The traceback of the error is as follows:

> Traceback (most recent call last):
>     result_aligned = whisperx.align(result["segments"], model_a, metadata, audio_path, device)
>   File ".../whisperx/alignment.py", line 301, in align
>     char_segments_arr = per_seg_grp.apply(lambda x: x.reset_index(drop = True)).reset_index()
>   File ".../pandas/core/frame.py", line 6217, in reset_index
>     new_obj.insert(
>   File ".../site-packages/pandas/core/frame.py", line 4780, in insert
>     raise ValueError(f"cannot insert {column}, already exists")
> ValueError: cannot insert subsegment-idx, already exists

When the error occurs, I attempt to try different models and re-transcribe but get the same error.

Please let me know if you require any additional information or clarification.

**Comment by pragmaware** [comment]
This is likely related to #98

**Comment by Barabazs** [comment]
Closed as resolved in #98
